pyat/at/lattice/cavity_access.py

```diff
@@ -286,9 +349,18 @@ def apply(ring: Lattice, cavpts, modif):
 Lattice.get_rf_frequency = get_rf_frequency
 Lattice.get_rf_timelag = get_rf_timelag
 Lattice.set_rf_voltage = set_rf_voltage
+Lattice.set_rf_frequency = set_rf_frequency
```

Comment:
You don't run into this import issue anymore? How did you fix it? I see that the physics module that you need to compute the revolution frequency is imported later....

Reply:
I don't import anything from the physics package:
```python
from .elements import RFCavity
from .lattice_object import Lattice
from .utils import AtError, make_copy, Refpts
```
All this is within the lattice package and previously imported in `lattice/__init__.py`.  The physics module is involved at run time (`cell_f = ring.get_revolution_frequency(**kwargs) * ring.periodicity`), as a `Lattice` method, after everything has been imported.

The point is that there must not be an import loop, which is not the case here.